packages/pf4-component-mapper/src/files/plain-text.js

```diff
@@ -4,11 +4,11 @@ import PropTypes from 'prop-types';
 import { TextContent, Text } from '@patternfly/react-core';
 import validTextFields from '@data-driven-forms/common/src/utils/valid-text-fields';
 
-const PlainText = ({ label, name, variant }) => (
-  <TextContent>
+const PlainText = ({ label, name, variant, TextProps, TextContentProps }) => (
```

Comment:
I think `TextProps` should be changed to `...rest` to make it consistent with the way we are spreading props in all other components. (`...rest` should be always spread to the main component.)

Just catch `component` props before you do it.